ckan/cli/cli.py

```diff
@@ -221,3 +221,6 @@ def ckan():
 ckan.add_command(less.less)
 ckan.add_command(generate.generate)
 ckan.add_command(user.user)
+
+if __name__ == "__main__":
```

Comment:
ping @alexandru-m-g - see https://github.com/ckan/ckan/discussions/6457#discussioncomment-1436466

Reply:
@teodorescuserban Can you explain a bit why this is needed ? We were running _ckan run_ before without needing this. Also, this change is not merged in CKAN core.

Reply:
I'll manage without it.